ImperatorToCK3/Source/Configuration/Configuration.cpp

```diff
@@ -49,7 +49,7 @@ void Configuration::registerKeys()
 		});
 	registerKeyword("ImperatorDeJure", [this](const std::string& unused, std::istream& theStream) {
 		const commonItems::singleString deJureString(theStream);
-		imperatorDeJure = IMPERATOR_DE_JURE(stoi(deJureString.getString()));
+		imperatorDeJure = static_cast<IMPERATOR_DE_JURE>(stoi(deJureString.getString()));
```

Comment:
Never ever trust stoi. Ever. Stoi hails from 7th circle of hell and you will eventually crash on it. While on that topic do not trust Fronter either, people may manually edit configuration and enter junk. Always wrap stoi in a try/catch block.

If I didn't do that in my work that's because I'm still learning.